package.json

```diff
@@ -24,12 +24,12 @@
     "react-test-renderer": "*"
   },
   "devDependencies": {
-    "babel-jest": "^20.0.3",
-    "babel-preset-react-native": "^2.1.0",
-    "jest": "^20.0.4",
-    "react": "16.0.0-alpha.12",
-    "react-native": "^0.46.1",
-    "react-test-renderer": "16.0.0-alpha.12"
+    "babel-jest": "^23.0.0",
+    "babel-preset-react-native": "4.0.0",
+    "jest": "*",
```

Comment:
Curious why these need to change? Setting dependencies to * can be dangerous.

Reply:
Donno, I must have played with something and forgot to revert 🤦‍♂️. I'll fix it this week and update the PR